enhancements/20220822-event-and-audit.md

```diff
@@ -0,0 +1,709 @@
+# Title
+
+Event and audit log 
```

Comment:
nit: I guess this should be moved to the `# Title`

Reply:
updated, thanks.